animation_nodes/nodes/fcurve/evaluate_fcurves_transforms.py

```diff
@@ -0,0 +1,44 @@
+import bpy
+from bpy.props import *
+from mathutils import Euler
+from ... base_types import AnimationNode
+from ... events import executionCodeChanged
+from ... utils.math import composeMatrix
+
+frameTypeItems = [
+    ("OFFSET", "Offset", "", 0),
+    ("ABSOLUTE", "Absolute", "", 1) ]
+
+class EvaluateFCurvesTransformsNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_EvaluateFCurvesTransformsNode"
+    bl_label = "Evaluate FCurves Transforms"
+
+    frameType: EnumProperty(
+        name = "Frame Type", default = "OFFSET",
+        items = frameTypeItems, update = executionCodeChanged)
```

Comment:
The update method should be `propertyChanged` now that we no longer change the execution code.